src/backend/Pg.h

```diff
@@ -180,6 +191,33 @@ class PgResult
             PQgetvalue(result_.get(), ntuple, nfield));
     }
 
+    /** Return field as equivalent to Postgres' BOOL type (8 bits).
+     *
+     * Note that this function does not guarantee that the result struct
+     * exists, or that the row and fields exist, or that the field is
+     * not null, or that the type is that requested.
+
+     * @param ntuple Row number.
+     * @param nfield Field number.
+     * @return Field contents.
+     */
+    bool
+    asBool(int ntuple = 0, int nfield = 0) const
+    {
+        std::string val = c_str(ntuple, nfield);
+        if (val == "true" || val == "t" || val == "yes" || val == "y" ||
+            val == "1")
```

Comment:
this seems...really dumb. but as far as I can tell these are the only legal values for `true` in pg, ditto with below for `false`.

Reply:
This does seem really dumb. I'm not super familiar w/ the postgres api, though

Can we just do 
```cpp
return val == "true"
    || val == "t"
    || val == "yes"
    || val == "y"
    || val == "1";
```

Is there every a case where we hit that throw block?